File: examples-sdk/python/701-mq-rabbitmq-pubsub.py
#!/usr/bin/env python3
"""701-mq-rabbitmq-pubsub — Python SDK equivalent
Compile: vil compile --from python --input 701-mq-rabbitmq-pubsub.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/701-mq-rabbitmq-pubsub.ts
#!/usr/bin/env tsx
// 701-mq-rabbitmq-pubsub — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 701-mq-rabbitmq-pubsub.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
